gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/src/main/java/io/gravitee/gateway/standalone/vertx/TimeoutServerResponse.java

```diff
@@ -81,18 +81,11 @@ public HttpHeaders trailers() {
         return response.write(buffer);
     }
 
-    @Override
-    public void end() {
-        response.end();
-
-        release();
-    }
 
     @Override
-    public void end(Buffer buffer) {
-        response.end(buffer);
-
+    public Response endHandler(Handler<Void> endHandler) {
         release();
+        return response.endHandler(endHandler);
```

Comment:
I think we should not call release when setting the endHandler but when calling end no ?